recipe/meta.yaml

```diff
@@ -38,21 +38,22 @@ requirements:
     # Correlations
     - phik >=0.11.1,<0.13
     # Examples
-    - requests >=2.24.0,<2.29
+    - requests >=2.24.0,<3
     # Progress bar
-    - tqdm >=4.48.2,<4.65
+    - tqdm >=4.48.2,<5
     - seaborn >=0.10.1,<0.13
-    - multimethod >=1.4,<1.10
+    - multimethod >=1.4,<2
     # metrics
-    - statsmodels >=0.13.2,<0.14
-    - typeguard >=2.13.2,<2.14
+    - statsmodels >=0.13.2,<1
+    # type checking
+    - typeguard >=2.13.2,<3
     - imagehash ==4.3.1
-    - pillow
```

Comment:
Why is pillow removed? Upstream are using `visions[type_image_path]` which pulls in `imagehash` and `pillow` and looking at the code, both seem to be used explicitly in the code.
* Pillow usage: https://github.com/search?q=repo%3Adylan-profiler%2Fvisions+PIL&type=code
* imagehash: https://github.com/dylan-profiler/visions/blob/dca86aa1f6be9d5872f7cce91414e1fa33bd1b85/src/visions/utils/images/image_utils.py#L5

Reply:
Thanks for catching this! I did not see it in the required dependencies in `setup.py`, so I was unsure if it was still required. I'll add it back.